### ISSUE TITLE
Few doubts about ProGran Implementation

### ISSUE THREAD

**Issue by chinmay5** [aladdinpersson/Machine-Learning-Collection#36]
Thank you so much for the wonderful implementation of the ProGan model. I just had a couple of doubts though-

1. In the WSConv2d [here](https://github.com/aladdinpersson/Machine-Learning-Collection/blob/37762890f0fc714f1f65b30e555fac4455b7f926/ML/Pytorch/GANs/ProGAN/model.py#L44), I am not really sure what is happening. Setting bias of Conv layer to None but storing its value before that. I am a bit confused.
2. Also [here](https://github.com/aladdinpersson/Machine-Learning-Collection/blob/37762890f0fc714f1f65b30e555fac4455b7f926/ML/Pytorch/GANs/ProGAN/train.py#L98), do we have some special reason for choosing a value of 30 for `PROGRESSIVE_EPOCHS` 

Would really appreciate your help.

**Comment by aladdinpersson** [comment]
Thanks! Regarding the first question I did it because we do not want to scale the bias term at each forward run so I manually copy it to a self.bias term inside the init method and then disable it inside the conv layer by setting `self.conv.bias = None`. There's no special reason for choosing a value of 30 for progressive epochs, the paper mentioned there should be 800k update steps as we increase alpha and then 800k update steps when alpha = 1. When I trained it I manually inspected before moving on to the next image resolution just to make sure they looked ok.
